src/MvcClient.Portal/Startup.cs

```diff
@@ -25,6 +25,8 @@ public void ConfigureServices(IServiceCollection services)
                     options.ResponseType = "code";
                     options.UsePkce = true;
                     options.ResponseMode = "query";
+                    options.SignedOutRedirectUri = "http://localhost:5002";
+                    options.SignedOutCallbackPath = "/";
 
```

Comment:
Probably don't need to set  options.SignedOutCallbackPath, and options.SignedOutRedirectUri should be set to https